view/frontend/web/js/view/payment/method-renderer/omise-offsite-installment-method.js

```diff
@@ -107,9 +206,28 @@ define(
              */
             getInstallmentTerms(id) {
                 const installmentBackends = checkoutConfig.installment_backends;
+                const templateLabel = $.mage.__('%terms months (%amount THB / month)');
+
                 for (const key in installmentBackends) {
-                    if (installmentBackends[key]._id === 'installment_' + id)
-                        return ko.observableArray(installmentBackends[key].allowed_installment_terms);
+                    if (installmentBackends[key]._id === 'installment_' + id) {
+                        const terms = installmentBackends[key].allowed_installment_terms;
+
+                        var dispTerms = [];
+                        for (let i = 0; i < terms.length; i++) {
```

Comment:
Simpler to use `forEach`?

Reply:
well maybe.. this is a habit from `nodejs`.. `foreach` takes way more resources than `for`..